homeassistant/components/homematic/sensor.py

```diff
@@ -251,6 +251,14 @@
         device_class=SensorDeviceClass.PM10,
         state_class=SensorStateClass.MEASUREMENT,
     ),
+    "LEVEL": SensorEntityDescription(
+        key="LEVEL",
+        native_unit_of_measurement=PERCENTAGE,
+    ),
+    "LEVEL2": SensorEntityDescription(
+        key="LEVEL2",
+        native_unit_of_measurement=PERCENTAGE,
+    ),
```

Comment:
Which devices would need that at all?

Reply:
Is `LEVEL2` correct? I think it must be `LEVEL_2`.